lib/scss_lint/cli.rb

```diff
@@ -54,9 +54,12 @@ def act_on_options(options)
     end
 
     def scan_for_lints(options, config)
+      files = FileFinder.new(config).find(options[:files])
+
       runner = Runner.new(config)
-      runner.run(FileFinder.new(config).find(options[:files]))
+      engines = runner.run(files)
```

Comment:
This is not an ideal approach. Calling `run` should not return a list of engines. The runner should auto-correct if auto-correction has been called for. There are a lot of other issues with the `Runner` class that are unrelated to your change, but it's purpose is to be the central coordinator for the tool. The `CLI` class is simply intended to parse CLI options and hand those off to the `Runner`.